tests/unit/test.errors.js

```diff
@@ -5,7 +5,7 @@ import {assert} from 'chai';
 import {
   onlyErrorsWithCode, isErrorWithCode, onlyInstancesOf,
 } from '../../src/errors';
-import {makeSureItFails} from './helpers';
+import {makeSureItFails, ErrorWithCode} from './helpers';
```

Comment:
since `ErrorWithCode` are used in two separate test files and seem potentially very useful, I'd leave them as it is